### PR TITLE
Cgt 3271

### PR DESCRIPTION
**Pull request by janeodoi**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -18,6 +18,10 @@ GET         /session-timeout        controllers.utils.TimeoutController.timeout
 GET         /sold-or-given-away                 controllers.SoldOrGivenAwayController.soldOrGivenAway
 POST        /sold-or-given-away                 controllers.SoldOrGivenAwayController.submitSoldOrGivenAway
 
+#Sold or Given Away
```
Comment (jameshforster) [inline conf/app.routes]: Please remove duplicate routes.

File: conf/app.routes [hunk 2]
```diff
@@ -96,10 +100,13 @@ GET         /improvements                       controllers.ImprovementsControll
 POST        /improvements                       controllers.ImprovementsController.submitImprovements
 
 #Disposal Date
-GET         /                                   controllers.DisposalDateController.disposalDate
```
Comment (jameshforster) [inline conf/app.routes]: Please restore deleted home route.

File: app/controllers/OutsideTaxYearController.scala [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+package controllers
+
+import controllers.predicates.ValidActiveSession
+import uk.gov.hmrc.play.frontend.controller.FrontendController
+
+object OutsideTaxYearController extends OutsideTaxYearController {
+
+}
+
+class OutsideTaxYearController extends FrontendController with ValidActiveSession {
```
Comment (jameshforster) [inline app/controllers/OutsideTaxYearController.scala]: Please make this into a trait.

**Comment by janeodoi** [comment]
corrected errors.

**Comment by janeodoi** [comment]
changes made

File: app/controllers/OutsideTaxYearController.scala (hunk 3 above)
Comment (janeodoi) [inline app/controllers/OutsideTaxYearController.scala]: changes made

File: conf/app.routes (hunk 2 above)
Comment (janeodoi) [inline conf/app.routes]: Changes made

File: conf/app.routes (hunk 1 above)
Comment (janeodoi) [inline conf/app.routes]: changes made

**Pull request merged**